cdisc-org/cdisc-rules-engine
# ISS-448: Add new rule type to support CG0014

Steps to test:

1. Run the unit test
2. Run a full validation
3. Run a test validation with the attached rule and dataset and ensure 1 error is reported.
[cg0014-test-data.zip](https://github.com/cdisc-org/cdisc-rules-engine/files/12254691/cg0014-test-data.zip)
